src/encoded/schemas/matrix_file.json

```diff
@@ -271,6 +271,48 @@
                     }
                 }
             }
+        },
+        "cell_mapping": {
+            "title": "Cell mapping information for this matrix",
+            "description": "Information at the raw matrix level that allows for mapping of cells in the final matrix to other entities in the experimental graph.",
```

Comment:
From this "...at the raw matrix level..." it sounds like this property is submitted for the MatrixFile object representing the raw matrix file, but we intend this to be at the final matrix, right?